crates/next-api/src/nft_json.rs

```diff
@@ -180,7 +180,7 @@ impl Asset for NftJsonAsset {
                 continue;
             }
 
-            if chunk == referenced_chunk.to_resolved().await? {
+            if chunk.eq(referenced_chunk) {
```

Comment:
Is there a reason to prefer `eq`?
```suggestion
            if chunk == referenced_chunk {
```

Reply:
`referenced_chunk` is a pointer so I could have deref'd it I guess